extensions/wikia/WallNotifications/WallNotifications.class.php

```diff
@@ -150,32 +153,45 @@ public function getWikiNotifications( $userId, $wikiId, $readSlice = 5, $counton
 	 * @return array
 	 */
 	public function getCounts( $userId ) {
-		global $wgMemc, $wgCityId;
-
-		wfProfileIn(__METHOD__);
-		$wikiList = $this->getWikiList( $userId );
+		return WikiaDataAccess::cache(
+			$this->getCacheKey( $userId ),
```

Comment:
Maybe this should be ```getCountsCacheKey```.   There's already a ```getCache``` method in this class that handles a different kind of caching.